File: go/internal/scoring/testdata/structural-v1/java/Rescan.java
final class Rescan {
  static int run(int[] values, int[] queries) {
    int found = 0;
    for (int query : queries) {
      for (int i = 0; i < values.length; i++) {
        if (values[i] == query) {
          found += i;
          break;
        }
      }
    }
    return found;
  }
}

File: go/internal/scoring/testdata/structural-v1/rust/rescan.rs
pub fn rescan(values: &[i32], queries: &[i32]) -> i32 {
    let mut found = 0;
    for query in queries {
        for (i, value) in values.iter().enumerate() {
            if value == query { found += i as i32; break; }
        }
    }
    found
}
